### PR TITLE
Drills

### PR DESCRIPTION
**Pull request by ncanderson**
Really enjoyed this exercise.

### CODE REVIEW

File: spec/deaf_grandma_spec.rb [hunk 1]
```diff
@@ -9,10 +9,18 @@
   end
   
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak("HI GRANDMA")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    if @bye_counter == 3
```
Comment (degoeym) [inline spec/deaf_grandma_spec.rb]: Not sure this is the way you'd want to run this test, but regardless, keep in mind that you need to test for more than one thing with yelling BYE to grandma.

File: lib/fizzbuzz.rb [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+    if (input % 3 == 0) && (input % 5 == 0) 
+      "FizzBuzz"
+  	elsif input % 5 == 0
+  	  "Buzz"
+    elsif input % 3 == 0
+  	  "Fizz"
+  	else
+  	  input
+  	end 
   end
```
Comment (degoeym) [inline lib/fizzbuzz.rb]: Please make your indentation for each if, elsif, and else consistent. Initially this made me think the first elsif would only hit if the first if evaluated to true for both checks.

Also, you don't need parenthesis around separate compare statements. Still works with them there, but I believe the convention is to not use them, especially for such small and simple comparisons.

File: spec/fizzbuzz_spec.rb [hunk 3]
```diff
@@ -9,14 +9,14 @@
...
 
   it "returns 'FizzBuzz' when input is divisible by 3 & 5" do
-    #implement your test here
+    expect(script.run(15)).to eq "FizzBuzz"
   end
   
   it "returns the input number when input isn't divisible by 3, 5, or both" do
-    #implement your test here
+    expect(script.run(22)).to eq 22
   end
```
Comment (degoeym) [inline spec/fizzbuzz_spec.rb]: Everything looks good here. :+1:

File: lib/fizzbuzz.rb (hunk 2 above)
Comment (nanderson83) [inline lib/fizzbuzz.rb]: Thanks for pointing out indentation errors - it's been mentioned before and
I though it was me not being careful, but I've updated my Sublime Text
settings and I hope my github repos will reflect what I am actually seeing
locally.

nate

Comment (degoeym) [inline lib/fizzbuzz.rb]: Gotcha. It's not a *huge* deal, but improperly formatted code tends to raise a developer's urge to kill. :stuck_out_tongue_closed_eyes: 

Also, have you checked out the Atom editor? It's what a lot of the cool kids are using now, and it's the listed editor on the Getting Started page.

http://paircolumbus.org/getting_started/

**Pull request closed without merging**